lib/shopify_api/resources/base.rb

```diff
@@ -4,6 +4,7 @@ module ShopifyAPI
   class Base < ActiveResource::Base
     class InvalidSessionError < StandardError; end
     extend Countable
+    self.format = :json
```

Comment:
Since this changes the default for all apps, it might break apps that currently do anything with the response body directly. This is an actual thing that developers do, unfortunately, so this change might break apps.

Since it is just to fix some tests, I wouldn't take the risk and find some other solution. If really necessary, we'll have to treat it as a breaking change, so that would mean a major version bump.